featuretools/primitives/standard/transform/binary/multiply_numeric_boolean.py

```diff
@@ -17,13 +17,14 @@ class MultiplyNumericBoolean(TransformPrimitive):
         the corresponding value in Y is True.
 
     Examples:
+        >>> import pandas as pd
         >>> multiply_numeric_boolean = MultiplyNumericBoolean()
         >>> multiply_numeric_boolean([2, 1, 2], [True, True, False]).tolist()
         [2, 1, 0]
         >>> multiply_numeric_boolean([2, None, None], [True, True, False]).tolist()
         [2.0, nan, nan]
-        >>> multiply_numeric_boolean([2, 1, 2], [True, True, None]).tolist()
-        [2.0, 1.0, nan]
+        >>> multiply_numeric_boolean([2, 1, 2], pd.Series([True, True, pd.NA], dtype="boolean")).tolist()
```

Comment:
With the new code the old boolean series had a dtype of object and pandas wasn't converting this to `Int64`. Updated here to use `boolean` since we should always get WW initialized series as inputs during normal CFM operations and that will have a dtype of either `bool` or `boolean`.